.github/workflows/labeler.yml

```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
+on: [pull_request]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
If the apache github organization has enabled Github Actions for all of the repos, this should just work. If not, I'll need to reach out to the ASF Infra team to have them enable github actions for this repo.